style.css

```diff
@@ -719,6 +771,49 @@ form {
   .hamburger:hover:disabled {
     background-color: #2230d2;
   }
+
+  .project-details-modal {
```

Comment:
- [ ] Let's increase the size of the image when the modal is open in desktop. This will help fit better the Figma Template.

Yours:
![image](https://user-images.githubusercontent.com/79114233/141168321-a1c860e5-52bf-4fb6-b19c-9597eb0bc7d0.png)

Figma Template:
![image](https://user-images.githubusercontent.com/79114233/141168347-cab30ad1-86aa-4453-8cf0-6582b3b1ae71.png)

Reply:
Thank you for your observation. I have fixed that